### ISSUE TITLE
It adds an almost 1 second latency to window swtiching

### ISSUE THREAD

**Issue by kantord** [termhn/i3-vim-nav#12]
Just by looking at the code, I don't see why it should be slow. Maybe the whole script takes a lot of time to initialize due to the dependencies? In that case it could perhaps by replaced by a server that keeps running?

**Comment by kantord** [comment]
There's BTW no noticeable lags within vim

**Comment by kantord** [comment]
Interestingly, there's no issue if I run it in the terminal:

```
[0] % time ~/.vim/plugged/i3-vim-nav/i3-vim-nav h
~/.vim/plugged/i3-vim-nav/i3-vim-nav h  0.06s user 0.06s system 87% cpu 0.133 total
```

**Comment by fu5ha** [comment]
Yeah I remember noticing this when I was actually using it too but I'm not sure why it is
